src/core/content/util.js

```diff
@@ -567,7 +567,10 @@ const Util = {
 		}
 
 		// Remove [genre] or 【genre】 from the beginning of the title
-		const title = videoTitle.replace(/^((\[[^\]]+])|(【[^】]+】))\s*-*\s*/i, '');
+		let title = videoTitle.replace(/^((\[[^\]]+])|(【[^】]+】))\s*-*\s*/i, '');
+
+		// Remove track (CD and vinyl) numbers from the beginning of the title
+		title = title.replace(/^\s*((([a-zA-Z]{1,2})|([0-9]{1,2}))[1-9]?\.\s+)/i, '');
```

Comment:
Would this simplify things? why so many braces? 

```suggestion
		title = title.replace(/^\s*([a-zA-Z]{1,2}|([0-9]{1,2})[1-9]?\.\s+/i, '');
```